src/qforte/ucc/spqe.py

```diff
@@ -180,7 +179,7 @@ def run(self,
                             sq_op.simplify()
                             self._pool_obj.add_term(0.0, sq_op)
                             self._excitation_dictionary[I] = idx
-                            self._excitation_indices.append(I)
+                            self._indices_of_zeroable_residuals.add(I)
```

Comment:
Minor suggestion. Maybe use "_indices_of_residuals_to_be_zeroed". The reasoning is that theyr could be other residuals that can be zeroed but are not included in the pool (especially now that we have a maximum excitation rank criterion).

Reply:
These aren't indices of residuals that can be zeroed. They are indices of residuals _that the algorithm could possibly enforce to be zero with the current pool, but not necessarily with the current ansatz_.

Reply:
I understand and I wrote the above suggestion knowing that. If we do not mind being more verbose, we can say "_indices_of_zeroable_residuals_in_pool". I believe that this would clear any possible confusion with zeroable residuals in general.